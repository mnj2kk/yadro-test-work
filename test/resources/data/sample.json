{
  "names": [
    "Алёна",
    "Миша",
    "Алёна",
    "Дима"
  ]
}