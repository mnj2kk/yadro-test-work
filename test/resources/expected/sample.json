{
  "count": 3,
  "valid": {
    "Дима": 1,
    "Алёна": 2,
    "Миша": 1
  }
}